views/gallery.hbs

```diff
@@ -49,4 +56,22 @@
 			</div>
 		</div>
 	</div>
-</div>
\ No newline at end of file
+</div>
+{{#section "script"}}
+<script>
+	function fallbackLoad(t){
+		$.ajax({url: t.attr("data-original"), headers: {"Range": "bytes=0-255"}, success: function(result){
```

Comment:
This has the potential to error on empty files with a 416 (Requested Range Not Satisfiable) error

Reply:
Yes it does return 416 on empty files. Should I add some kind of special handling for those? Currently it just don't show anything.

Reply:
Maybe some "Empty file" or "preview not shown" text?